process/coredump.go

```diff
@@ -312,6 +312,12 @@ func (cd *CoredumpProcess) OpenELF(path string) (*pfelf.File, error) {
 	return nil, fmt.Errorf("ELF file `%s` not found", path)
 }
 
+// ExtractAsFile implements the Process interface
```

Comment:
```suggestion
// ExtractAsFile implements the Process interface.
```

Reply:
This follows style of the file. Would it make sense to fixup the whole file in a follow up PR?